src/sentry/api/endpoints/organization_events_facets_stats_performance.py

```diff
@@ -93,31 +84,96 @@ def get_event_stats(
                     # but we generally only need pretty low granularity
                     # for this since it's only being used for sparklines
-                    rollup=ONE_DAY,
+                    rollup=SIX_HOURS,
                     limit=10000,
                     organization=None,
                     referrer=referrer,
                 )
 
-        results = self.get_event_stats_data(
-            request,
-            organization,
-            get_event_stats,
-            top_events=5,
-            query=filter_query,
-            query_column="count()",
-            additional_query_column="p75(transaction.duration)",
-        )
+            results = self.get_event_stats_data(
+                request,
+                organization,
+                top_events_stats,
+                top_events=5,
+                query=filter_query,
+                query_column="count()",
+                additional_query_column="p75(transaction.duration)",
+            )
+
+            def get_event_stats(
+                query_columns: Sequence[str],
+                query: str,
+                params: Dict[str, str],
+                rollup: int,
+                zerofill_results: bool,
+                comparison_delta: Optional[datetime] = None,
+            ):
+                return discover.timeseries_query(
+                    selected_columns=query_columns,
+                    query=query,
+                    params=params,
+                    # TODO: Better selection of granularity,
+                    # but we generally only need pretty low granularity
+                    # for this since it's only being used for sparklines
+                    rollup=SIX_HOURS,
+                )
+
+            events_stats = self.get_event_stats_data(
+                request,
+                organization,
+                get_event_stats,
+                top_events=5,
+                query=filter_query,
+                query_column="p75(transaction.duration)",
+            )
 
-        totals = {}
-        for facet in top_facets["data"]:
-            key = facet.pop("tags_key")
-            value = facet.pop("tags_value")
-            totals[f"{key},{value}"] = facet
+        with sentry_sdk.start_span(op="discover.endpoint", description="find_correlation"):
+            totals = {}
+            for facet in top_facets["data"]:
+                key = facet.pop("tags_key")
+                value = facet.pop("tags_value")
+                new_key = f"{key},{value}"
 
-        results["totals"] = totals
+                sum_correlation = corr_snuba_timeseries(
+                    results[new_key]["count()"]["data"], events_stats["data"]
+                )
+                facet["sum_correlation"] = sum_correlation
+                totals[new_key] = facet
+
+            results["totals"] = totals
 
         return Response(
             results,
             status=200,
         )
+
+
+def corr_snuba_timeseries(
```

Comment:
I wonder if this would fit better in discover.py since that (for now) has a bunch of util functions, like zerofill?